inventory/multi_inventory.py

```diff
@@ -223,7 +236,7 @@ def get_inventory(self):
                               ]
                     raise RuntimeError('\n'.join(err_msg).format(**result))
                 else:
-                    self.all_ec2_results[result['name']] = json.loads(result['out'])
+                    self.all_inventory_results[result['name']] = json.loads(result['out'])
```

Comment:
We need to check to ensure that result['name'] is unique. Check needs to happen when we read in the config file.